test/time_tracker/timers/pubsub_test.clj

```diff
@@ -1,6 +1,7 @@
 (ns time-tracker.timers.pubsub-test
   (:require [clojure.test :refer :all]
             [clojure.core.async :refer [chan alt!! put!] :as async]
+            [clojure.string :as str]
```

Comment:
There is a `clojure.core/str`. Don't shadow stuff from `clojure.core`.
You can use `(:refer-clojure :exclude [str])` if you are not interested in `clojure.core/str`. But I recommend requiring this as something else, like simply `string`.